nzedb/Contents.php

```diff
@@ -27,6 +29,9 @@ public function __construct(array $options = [])
 		$this->pdo = ($options['Settings'] instanceof DB ? $options['Settings'] : new DB());
 	}
 
+	/**
+	 * @return array|bool
```

Comment:
Should have a description. Just the type doesn't help the coder, they still have to come to the code and read it to find out what it is.